src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,30 @@
 import React from 'react';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+import { Comment } from '../../types';
+
+type Props = {
+  comment: Comment;
+};
```

Comment:
You can move the types to a separate Type file for better clarity and readability